integreat_cms/api/v3/location_categories.py

```diff
@@ -30,6 +31,9 @@ def transform_location_category(location_category, language_slug):
         else location_category.name,
         "color": location_category.color,
         "icon": location_category.icon,
+        "icon_url": static(
+            "/svg/poi-category-icons/" + location_category.icon + ".svg"
```

Comment:
```suggestion
        "icon_url": (
            settings.BASE_URL
            + static(f"/svg/poi-category-icons/{location_category.icon}.svg")
```
1. We need the base url, since the web app is on a different server and requesting `/static/...` won't return the correct file
2. f-strings please? :point_right::point_left: